content/ember/v3/router-events.md

```diff
@@ -45,18 +45,19 @@ import Router from '@ember/routing/router';
 import { inject as service } from '@ember/service';
 
 export default Router.extend({
+  router: service(),
```

Comment:
@JoshCrozier thank you for the PR! 

I wonder if it's appropriate to inject the Router service into an extension of the Router here? 

I could see how we might want to inject the Router service if we were on a Route as is done [here](https://github.com/emberjs/ember.js/pull/17553/files), but that doesn't seem to be the case in these examples.

What do you think?